packages/block-library/src/social-links/edit.js

```diff
@@ -22,16 +27,33 @@ const TEMPLATE = [
 	[ 'core/social-link', { service: 'youtube' } ],
 ];
 
-export function SocialLinksEdit() {
+export function SocialLinksEdit( props ) {
+	const {
+		attributes: { openInNewTab },
+		setAttributes,
+	} = props;
 	return (
-		<InnerBlocks
-			allowedBlocks={ ALLOWED_BLOCKS }
-			templateLock={ false }
-			template={ TEMPLATE }
-			orientation="horizontal"
-			__experimentalTagName={ Block.ul }
-			__experimentalAppenderTagName="li"
-		/>
+		<Fragment>
+			<InspectorControls>
+				<PanelBody title={ __( 'Link Settings' ) }>
+					<ToggleControl
+						label={ __( 'Open Links in new Tab' ) }
```

Comment:
Minor change to sentence case

```suggestion
						label={ __( 'Open links in new tab' ) }
```